pkg/controllers/updaterun/execution.go

```diff
@@ -314,6 +327,24 @@ func (r *Reconciler) checkAfterStageTasksStatus(ctx context.Context, updatingSta
 	return true, nil
 }
 
+// updateBindingRolloutStarted updates the binding status to indicate the rollout has started.
+func (r *Reconciler) updateBindingRolloutStarted(ctx context.Context, binding *placementv1beta1.ClusterResourceBinding) error {
+	cond := metav1.Condition{
+		Type:               string(placementv1beta1.ResourceBindingRolloutStarted),
+		Status:             metav1.ConditionTrue,
+		ObservedGeneration: binding.Generation,
+		Reason:             condition.RolloutStartedReason,
+		Message:            "Detected the new changes on the resources and started the rollout process",
+	}
+	binding.SetConditions(cond)
```

Comment:
do you care about "LastTransitionTime"?  as the binding status is most likely already True so it will not be updated.

Reply:
Good catch! I didn't notice this during test. Fixed, thanks!